README.md

```diff
@@ -87,9 +87,8 @@ need them.
 
 ### Platform support
 
-While Rustls itself is platform independent, by default it uses
-[`ring`](https://crates.io/crates/ring) for implementing the cryptography in
-TLS. As a result, rustls only runs on platforms
+While Rustls itself is platform independent, by default it uses [`ring`] for implementing
+the cryptography in TLS. As a result, rustls only runs on platforms
```

Comment:
The " We do not presently support WebAssembly" isn't clear enough. It would be good to state that *ring* does support WebAssembly, and probably it would be good to mention why Rustls doesn't.

Reply:
Please open a new PR with the wording you'd prefer to see w.r.t web assembly compatibility. This is pre-existing text that isn't being adjusted in this branch.